### PR TITLE
Step 3: Fetched array list

### PR DESCRIPTION
**Pull request by gracetian6**
Step 3 was about JSON. The task is to modify DataServlet.java to return some hard-coded JSON data, and then modify script.js to fetch that JSON and build a UI from it. 
Demo: https://drive.google.com/file/d/1yRpmGH3QVj5JLWn1Vj95hVrE1n-2zHgl/view?usp=sharing

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -19,14 +19,34 @@
...
+  private final ArrayList<String> msgs = new ArrayList<String>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    msgs.add("hello");
+    msgs.add("there");
+    msgs.add("world");
+    System.out.println(msgs);
```
Comment (jessicaslaughter) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: now that you confirmed this works, you can remove the system.out.println statements

**Review by jessicaslaughter — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -19,14 +19,34 @@
...
+    String json = convertToJson(msgs);
+    System.out.println(msgs);
```
Comment (jpnickolas) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Remove debug statements

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -19,14 +19,34 @@
...
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    msgs.add("hello");
+    msgs.add("there");
+    msgs.add("world");
+    System.out.println(msgs);
+    String json = convertToJson(msgs);
```
Comment (jpnickolas) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: use `gson.toJson(msgs);` instead of writing your own helper.

**Review by jpnickolas — CHANGES_REQUESTED** [review]

**Review by jpnickolas — APPROVED** [review]
Looks good Grace!

**Pull request merged**